runtime/onert/core/src/ir/train/TrainableGraph.cc

```diff
@@ -125,9 +126,61 @@ const ITrainableOperation &TrainableGraph::operation(OperationIndex index) const
   return dynamic_cast<const ITrainableOperation &>(_graph.operations().at(index));
 }
 
+void TrainableGraph::assertValidTopologicalOrder(const std::vector<ir::OperationIndex> &order,
+                                                 const std::string &order_type = "forward") const
```

Comment:
Why did you use string type argument? IMO, we can use boolean type for this.

Reply:
Updated.